src/driver-steelseries.c

```diff
@@ -625,8 +651,13 @@ steelseries_write_led_v1(struct ratbag_led *led)
 			buf[2] = (led->brightness / 86) + 2;
 		}
 	} else {
-		buf[0] = STEELSERIES_ID_LED_COLOR_SHORT;
-		buf[1] = led->index + 1;
+		if (alternate_led) {
+			buf[0] = 0x05;
```

Comment:
All the other commands are added as constants in the beginning, should it be done the same way here? If so, should I name it "STEELSERIES_ID_LED_COLOR_SHORT_ALTERNATE" or somehow different?